src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -86,6 +86,7 @@
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.Scripting.Destkop.UnitTests" />
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.CSharp.Scripting.UnitTests" />
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.CSharp.Scripting.Desktop.UnitTests" />
+    <InternalsVisibleToTest Include="Roslyn.InteractiveHost.UnitTests" />
```

Comment:
@sharwell Since you asked for approval for *InternalsVisibleToTest* in #24460 you may also want approval for this one.

Reply:
👍 This one should be covered under the umbrella of the review from @jaredpar.